### ISSUE TITLE
TOPPView error when clicking protein accession

### ISSUE THREAD

**Issue by cbielow** [OpenMS/OpenMS#5798]
[Downloads.zip](https://github.com/OpenMS/OpenMS/files/8025528/Downloads.zip)

Opening the mzML + annotating IDs from the attached zip works fine, but then clicking on the protein accession in the id view tab, an exception is thrown.
![image](https://user-images.githubusercontent.com/6008722/153028557-59bbdc9f-a9fc-4115-9960-0228c6195c00.png)

**Comment by jpfeuffer** [comment]
AFAIK this only supports sp and tr accessions right now

**Comment by timosachsenberg** [comment]
I can provide a quick fix (e.g., catch the exception).

**Comment by jpfeuffer** [comment]
In theory this should be activated for all accessions. You could still have a sequence annotated in the idXML. The exception should be caught on the JS side if no sequence is there and no accession can be found on uniprot.

**Comment by jpfeuffer** [comment]
Ah I confused the sequence click.

**Comment by jpfeuffer** [comment]
Yeah I am not sure how we can decide on TOPPView C++ side if an accession can be shown. Maybe the sp/tr check is the only way. In this case it would even be better if those accessions weren't underlined+blue.
